manager/src/main/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPI.java

```diff
@@ -34,7 +34,7 @@
 public class ShardBridgesSyncAPI {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ShardBridgesSyncAPI.class);
-    private static final List<BridgeStatus> statuses = Arrays.asList(BridgeStatus.REQUESTED, BridgeStatus.PROVISIONING, BridgeStatus.DELETION_REQUESTED);
```

Comment:
Isn't this change just hiding in all the endpoints bridges in `PROVISIONING` status? Why this change? Can't this happen in real world?

Reply:
Yep it can happen in the real-world, but not really in the current implementation given that everything is tied into a single deployment and we just start from fresh whenever we deploy the components. 

As/when we start to split things out, we support restarting of components and also look into correct failing handling, then we'll need to revisit what is returned from this API.

Reply:
Thanks for the clarification, do we have a ticket for that? So that we keep this in the backlog